common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -86,55 +86,56 @@ public class CommonValue {
-                        + "   \"" + MLModelGroup.ACCESS + "\": {\n"
-                        + "      \"type\": \"keyword\"\n"
-                        + "    },\n"
-                        + "    \"" + MLModelGroup.OWNER + "\": {\n"
-                        + "      \"type\": \"nested\",\n"
-                        + "        \"properties\": {\n"
-                        + "          \"name\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\", \"ignore_above\":256}}},\n"
-                        + "          \"backend_roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
-                        + "          \"roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
-                        + "          \"custom_attribute_names\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}}\n"
-                        + "        }\n"
-                        + "    },\n"
-                        + "     \"" + MLModelGroup.CREATED_TIME_FIELD + "\": {\n"
-                        + "      \"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
-                        + "    \"" + MLModelGroup.LAST_UPDATED_TIME_FIELD + "\": {\n"
-                        + "      \"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"}\n"
-                        + "  }\n"
-                        + "}";
+                + "  \"_meta\": {\n"
+                + "    \"schema_version\": " + ML_MODEL_GROUP_INDEX_SCHEMA_VERSION + "\n"
+                + "  },\n"
+                + "  \"properties\": {\n"
+                + "    \"" + MLModelGroup.MODEL_GROUP_NAME_FIELD + "\": {\n"
+                + "      \"type\": \"text\",\n"
+                + "      \"fields\": {\n"
+                + "        \"keyword\": {\n"
+                + "          \"type\": \"keyword\",\n"
+                + "          \"ignore_above\": 256\n"
+                + "        }\n"
+                + "      }\n"
+                + "    },\n"
+                + "    \"" + MLModelGroup.DESCRIPTION_FIELD + "\": {\n"
+                + "      \"type\": \"text\"\n"
+                + "    },\n"
+                + "    \"" + MLModelGroup.LATEST_VERSION_FIELD + "\": {\n"
+                + "      \"type\": \"integer\"\n"
+                + "    },\n"
+                + "    \"" + MLModelGroup.MODEL_GROUP_ID_FIELD + "\": {\n"
+                + "      \"type\": \"keyword\"\n"
+                + "    },\n"
+                + "    \"" + MLModelGroup.BACKEND_ROLES_FIELD + "\": {\n"
+                + "      \"type\": \"text\",\n"
+                + "      \"fields\": {\n"
+                + "        \"keyword\": {\n"
+                + "          \"type\": \"keyword\",\n"
+                + "          \"ignore_above\": 256\n"
+                + "        }\n"
+                + "      }\n"
+                + "    },\n"
+                + "    \"" + TENANT_ID + "\" : {\"type\": \"keyword\"},\n"
```

Comment:
This works 'cause it's valid JSON, but all the other strings (see `MlModelGroup.ACCESS` right below) do the "pretty" JSON equivalent.  Might want to be consistent.

Reply:
```
{
    "took": 11,
    "timed_out": false,
    "_shards": {
        "total": 1,
        "successful": 1,
        "skipped": 0,
        "failed": 0
    },
    "hits": {
        "total": {
            "value": 1,
            "relation": "eq"
        },
        "max_score": 1.0,
        "hits": [
            {
                "_index": ".plugins-ml-model-group",
                "_id": "k7BBoJEBmupm26gNRc4D",
                "_score": 1.0,
                "_source": {
                    "name": "Cohere embedding normal",
                    "latest_version": 1,
                    "description": "embedding sagemaker model",
                    "access": "public",
                    "created_time": 1724970517643,
                    "last_updated_time": 1724970517930,
                    "tenant_id": "1234567"
                }
            }
        ]
    }
}
```

This is mapping I got after making this change. Seems like similar?